_data/developer.yml

```diff
@@ -33,8 +33,9 @@ websites:
       url: https://bitbucket.org
       twitter: bitbucket
```

Comment:
And thank you. It all looks good. Just waiting for another collaborator to approve before merge.